resources/assets/icons/status/fatal.svg

```diff
@@ -0,0 +1 @@
+<svg version="1.1" id="icon" xmlns="http://www.w3.org/2000/svg" x="0" y="0" viewBox="0 0 20 20" xml:space="preserve"><path d="M10 19c5 0 9-4 9-9s-4-9-9-9-9 4-9 9h0c0 5 4 9 9 9zm-6.4-2.6L16.4 3.6" fill="none" stroke="currentColor" stroke-width="2" stroke-linecap="round" stroke-linejoin="round"/></svg>
```

Comment:
identical to 
![image](https://user-images.githubusercontent.com/35610748/128482335-0e93c994-3a19-474c-9f27-8d24bac5213d.png)

Reply:
@ItsANameToo this one is not equal, the blocked one has wider lines

Reply:
blocked:
![image](https://user-images.githubusercontent.com/17262776/128522872-cd6d12ec-8262-4a13-ac54-0c86f1aea06b.png)
Fatal:
![image](https://user-images.githubusercontent.com/17262776/128522901-840d223a-389d-4156-9a43-9e316bd99364.png)

Reply:
then it's up to oleg to define which one to use, they look identical otherwise 🤔

Reply:
@ItsANameToo the one that I added is the only one that looks good, the one for `notifications/blocked` looks weird (same for the clock icon below)

![image](https://user-images.githubusercontent.com/17262776/128523323-ac8b06d9-9d7b-417d-87c1-73a8a518dc9a.png)

Reply:
FYI I asked Oleg about which one to use